tests/functional/audit/test_update_rbac_objects.py

```diff
@@ -71,12 +71,59 @@ def new_rbac_objects_info(sdk_client_fs) -> Dict[str, Dict[str, Dict]]:
     }
 
 
+@pytest.fixture()
+def prepared_changes(sdk_client_fs, rbac_create_data, new_rbac_objects_info) -> dict:
+    """
+    Prepare dict with "obejct_changes"
+    """
+    c = sdk_client_fs
+    initial = rbac_create_data
+
+    def _get(key1, key2):
+        return new_rbac_objects_info[key1]['correct'][key2]
+
+    return {
+        'user': {
```

Comment:
Let's start using double quotes ?

Reply:
I'll try, but there's still a lot of this in functional tests